DateToolsSwift/DateTools/DateTools.bundle/ko.lproj/DateTools.strings

```diff
@@ -26,16 +26,16 @@
 "An hour ago" = "1시간 전";
 
 /* No comment provided by engineer. */
-"Just now" = "방금 전";
```

Comment:
There could be discrepancy in whether 방금 is more commonly used or 방금 전 is more commonly used. 전 means
 ago. 방금(A short time ago) already has the meaning 'ago', but according to the National Korean Language Institute, there has been use cases of 방금 전 being used in real life. 
http://www.korean.go.kr/front/mcfaq/mcfaqView.do?mn_id=62&mcfaq_seq=2561

This could be more of a preference choice, so... Do what you will